core/internal/cltest/cltest.go

```diff
@@ -376,7 +376,7 @@ func NewApplicationWithConfig(t testing.TB, cfg *configtest.TestGeneralConfig, f
 		}
 	}
 	if lggr == nil {
-		lggr = cfg.CreateProductionLogger()
+		lggr = cfg.CreateProductionLogger().Named(t.Name())
```

Comment:
Any objections to attaching the test name by default? It's a little noisy but helps with interleaved parallel tests.
Can follow up in this story next too: https://app.shortcut.com/chainlinklabs/story/15902/improve-logger-usage-in-tests